sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -115,8 +117,11 @@ def _setup_new_types_env(self):
     table_data = [
         {'bytes':b'xyw=', 'date':'2011-01-01', 'time':'23:59:59.999999'},
         {'bytes':b'abc=', 'date':'2000-01-01', 'time':'00:00:00'},
-        {'bytes':b'dec=', 'date':'3000-12-31', 'time':'23:59:59.990000'}
+        {'bytes':b'dec=', 'date':'3000-12-31', 'time':'23:59:59.990000'},
+        {'bytes':b'\xab\xac\xad', 'date':'2000-01-01', 'time':'00:00:00'}
     ]
+    for row in table_data:
```

Comment:
Let's add a comment here that a bigquery client requires the user to base64-encode bytes sequences before passing them to BQ. If you have a link handy, please include it. 
Thank you.